scripts/exregional_point_source.sh

```diff
@@ -162,7 +164,7 @@ POST_STEP
 #
 #-----------------------------------------------------------------------
 #
-mv_vrfy "${DATA}/PT" ${INPUT_DATA}
+#mv_vrfy "${DATA}/PT" ${INPUT_DATA}
 
```

Comment:
@bbakernoaa, if this part and above, which were commented out, are not necessary, please remove them from the script.